### PR TITLE
Fixer mixer

### PR DESCRIPTION
**Pull request by crStiv**
hi there, just fixed multiple typos, hope it'll be useful for you :)

### CODE REVIEW

**Comment by donosonaumczuk** [comment]
Hey @crStiv thanks for your contribution. The branch that has the latest changes is `development`, I edited your PR to point there. Do you want to update it?

**Comment by crStiv** [comment]
> Hey @crStiv thanks for your contribution. The branch that has the latest changes is `development`, I edited your PR to point there. Do you want to update it?

I don't see the "Update branch" button. Is my branch already synchronized with development?

**Comment by crStiv** [comment]
> Hey @crStiv thanks for your contribution. The branch that has the latest changes is `development`, I edited your PR to point there. Do you want to update it?

Yep, sure

**Comment by donosonaumczuk** [comment]
> I don't see the "Update branch" button. Is my branch already synchronized with development?

No, it has conflicts, many of the comments you fixed were removed because were TODOs

**Comment by crStiv** [comment]
> > I don't see the "Update branch" button. Is my branch already synchronized with development?
> 
> No, it has conflicts, many of the comments you fixed were removed because were TODOs

Oh I see, so what do you recommend me to do then?

File: contracts/dashboard/primitives/app/AppCore.sol [hunk 1]
```diff
@@ -15,7 +15,7 @@ library AppCore {
     // Storage
 
     struct Storage {
-        string metadataURI; // Name, description, logo, other attribiutes like category/topic, etc.
+        string metadataURI; // Name, description, logo, other attributes like category/topic, etc.
```
Comment (donosonaumczuk) [inline contracts/dashboard/primitives/app/AppCore.sol:18]: This is the only relevant change when you compare against development branch

**Comment by donosonaumczuk** [comment]
Thanks, added your username as co-author
https://github.com/lens-protocol/lens-v3/commit/a4e9313abdc0a39da222e97d074c27ccdf58b6a8

**Pull request closed without merging**